Dockerfile

```diff
@@ -14,14 +14,14 @@ ENV TERM xterm
 
 ENV TERM xterm
 
+#The solr config file needs to be added after installation or it fails.
+ADD docker_files/solr /etc/default/solr.docker
```

Comment:
Hi Andy, thank you for the explanation and colaboration! I am merging your pull request :)